.gitignore

```diff
@@ -30,3 +30,7 @@ hs_err_pid*
 
 .idea
 target
+
+# OS generated files
+.DS_Store
+**/.DS_Store
```

Comment:
GitHub warns there is no newline at the end of the file. Could you add it?

Reply:
☑️ Done!